lib/spack/spack/build_systems/python.py

```diff
@@ -91,6 +92,16 @@ def configure(self, spec, prefix):
     # Default phases
     phases = ['build', 'install']
 
+    # Name of modules that the Python package provides
+    # This is used to test whether or not the installation succeeded
+    # These names generally come from running:
+    #
+    # >>> import setuptools
+    # >>> setuptools.find_packages()
+    #
+    # in the source tarball directory
```

Comment:
Is there some way to associate a docstring with a particular attribute in a class? I would like this information to show up in the API docs.